honeybee_schema/radiance/material.py

```diff
@@ -0,0 +1,280 @@
+"""Material Schema"""
+from pydantic import Field, constr
+from typing import List, Union
+
+from .._base import IDdBaseModel
+
+
+class Void(IDdBaseModel):
+    """Void modifier"""
+
+    type: constr(regex='^Void$') = 'Void'
+
+
+class ModifierBase(IDdBaseModel):
+    """Base class for Radiance Modifiers"""
+
+    # TODO: We can't define union of schema objects that haven't been
+    # defined yet. We also can't move this down, because ModifierBase
+    # is inherited by all these objects.
+    modifier: Union[
+         Void, Plastic, Glass, BSDF, Glow, Light, Trans
+    ] = Field(
+        # TODO: This needs to be fixed. How can we add add default=Void,
+        # when Void is not serializable?
+        default='void',
```

Comment:
You should just set this to be an instance of `Void`. This should be similar to how we use a simple `Autocalculate` object for a lot of the energy properties:
https://github.com/ladybug-tools/honeybee-schema/blob/master/honeybee_schema/bc.py#L24